pkg/agent/openflow/fields.go

```diff
@@ -22,46 +22,75 @@ import (
 // Fields using reg.
 var (
 	// reg0 (NXM_NX_REG0)
-	// reg0[0..3]: Field to mark the packet source. Marks in this field include,
-	//   - 0: from the tunnel port
-	//   - 1: from antrea-gw0
-	//   - 2: from the local Pods
-	//   - 4: from the Bridge interface
-	//   - 5: from the uplink interface
+	// reg0[0..3]: Field to store the packet source. Marks in this field include,
+	//   - 0: from tunnel interface.
```

Comment:
if there are more tunnel interfaces added eg: for multicluster, is it OK to use the same '0'? or we need define a new value here? @hongliangl @wenyingd

Reply:
I think this is a big issue and we need to discuss it in detail.